pkg/database/database.go

```diff
@@ -7,17 +7,24 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+
 	"github.com/kubearchive/kubearchive/pkg/models"
 	_ "github.com/lib/pq"
 )
 
 const (
-	resourceTableName = "resource"
-	resourcesQuery    = "SELECT data FROM %s WHERE kind=$1 AND api_version=$2"
+	resourceTableName   = "resource"
+	resourcesQuery      = "SELECT data FROM %s WHERE kind=$1 AND api_version=$2"
+	resourceIdQuery     = `SELECT id FROM %s WHERE kind=$1 AND api_version=$2 AND cluster=$3 AND cluster_uid=$4 AND name=$5 AND namespace=$6`
+	updateResourceQuery = `UPDATE %s SET resource_version=$1, updated_ts=$2, cluster_deleted_ts=$3, data=$4 WHERE id=$5`
```

Comment:
I think other fields need to be updated as well. What if the name changed? Or the namespace?

Reply:
added them